### DISCUSSION TITLE
Do you think I would be able to convert this WP site to headless and NextJS with this?

### DISCUSSION THREAD

**Discussion by RangerCoder99** [colbyfayock/next-wordpress-starter#201] [Q&A]
I wonder if it would work with joemygod.com that has A LOT of blog posts and add 5/10 every hour?  I'm trying to put this site on Next/Gatsby for years! Keep running into these major issues:

It uses existing DISQUS settings and DISQUS comments that can't be lost!
It uses the /2021/06/article name/ and got archive pages on the year and year/month part
It uses a ton of YouTube, Twitter, and other social media embeds 

The home page pulls all blog posts with number pages, pagination what the sample already seems to do.

Apart from that, it's your standard WordPress site 

Do you think it's possible?

**Comment by colbyfayock** [comment]
hey @RangerCoder99 , sorry i missed coming back to this

The only thing tricky about high volume and constantly changing sites is static deployments, if you're deploying that way, can take a while. However, there are 2 solutions around that, where with Netlify, you can use [Distributed Persistent Rendering](https://www.netlify.com/blog/2021/04/14/distributed-persistent-rendering-a-new-jamstack-approach-for-faster-builds/) which would only build a page when accessed or with Vercel you have [Incremental Static Regeneration](https://nextjs.org/docs/basic-features/data-fetching#incremental-static-regeneration)

you could also consider taking advantage of serverside if thats a route you'd prefer to take

**Comment by RangerCoder99** [comment]
For some reason Github did not let me mark it as the answer I just get the Github helpful tooltip popup that I can't close no matter where I click. But no worries about missing it before... in the end, we did go with Vercel.
